CHANGELOG.md

```diff
@@ -13,12 +13,11 @@
 * Do not remove field from the form when its value is an empty string. Fixes UIIN-2787.
 * Update Permission name for Inventory: Set records for deletion and staff suppress. Refs UIIN-2855.
 * ECS Member tenant - add `Shared` and `Held by` facets for the `Classification` browse. Refs UIIN-2813.
-* Jest/RTL: Cover CalloutRenderer component with unit tests. Refs UIIN-2665.
+* Jest/RTL: Cover ImportRecord component with unit test. Refs UIIN-2667.
 
 ## [11.0.3] (IN PROGRESS)
 
 * Fetch facets with the same query as `Contributor` records when the search is done by selecting browse result. Refs UIIN-2859.
-* Change the `Effective location` call numbers facet query to not see locations assigned to items with an empty `Effective call number` field. Refs UIIN-2871.
```

Comment:
Do you really need to remove previously added information in CHANGEOG file?

Reply:
Fixed